packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -172,12 +172,13 @@ Conflicts: %{_cross_os}image-feature(no-fips)
 rpmkeys --import %{S:1} --dbpath "${PWD}/rpmdb"
 rpmkeys --checksig %{S:0} --dbpath "${PWD}/rpmdb"
 rm -rf "${PWD}/rpmdb"
-rpm2cpio %{S:0} | cpio -iu linux-%{version}.tar config-%{_cross_arch} "*.patch"
-tar -xof linux-%{version}.tar; rm linux-%{version}.tar
+rpm2cpio %{S:0} | cpio -iu linux-%{version}.tar.xz config-%{_cross_arch} "*.patch" kernel.spec
+tar -xof linux-%{version}.tar.xz; rm linux-%{version}.tar.xz
+readarray -t patches < <(grep -P "^Patch\d+: .*.patch" kernel.spec | awk '{ print $2 }')
```

Comment:
Nit: a note on why we are doing this will be helpful

Reply:
Good point, I'll add a comment